.github/workflows/release_draft.yml

```diff
@@ -32,35 +32,36 @@ jobs:
       - name: Validate format of received tag
         uses: actions/github-script@v7
         with:
```

Comment:
This can be commented out as well

```suggestion
#      - name: Validate format of received tag
#        uses: actions/github-script@v7
#        with:
```